storageconnector/gitstorage_test.go

```diff
@@ -4,18 +4,23 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"regexp"
 	"testing"
 
 	"gopkg.in/src-d/go-git.v4"
+	"github.com/zalando-incubator/pazuzu/shared"
+	"github.com/davecgh/go-spew/spew"
 )
 
 var (
 	testRepository = filepath.Join("fixtures", "git")
-	storage        StorageReader
+	storage StorageReader
 )
 
 func TestMain(m *testing.M) {
+	spew.Config = spew.ConfigState{
+		DisableCapacities: true,
+		DisablePointerAddresses: true,
+	}
```

Comment:
Do we really need an extra dependency? Assert library does quite good pretty printing.